package.json

```diff
@@ -90,6 +90,7 @@
   },
   "scripts": {
     "start": "node --max-old-space-size=2048 node_modules/.bin/start-storybook -p 9001 -c .storybook",
+    "watch": "TSC_NONPOLLING_WATCHER=1 lerna run watch --parallel --stream",
```

Comment:
`TSC_NONPOLLING_WATCHER` uses OS-level directory change events. Without this, polling is used and it takes up a lot of CPU (thousands of files are polled for changes). This might now work for every OS.

The CPU is made worse by how this project is set up - the same file is watched multiple times. For instance, the `Button` module is used by many other modules and so it and its dependencies are watched multiple times. If the project was re-architected to have 1 build command in the base `package.json`, this issue would be mitigated and possible build times could be improved (120s build times currently).